app/components/Utility/BindToChainState.jsx

```diff
@@ -73,9 +77,11 @@ function BindToChainState(Component, options = {}) {
                 this.chain_key_refs = [];
                 this.chain_address_balances = [];
                 this.chain_assets = [];
+                this.chain_liquidity_pools = [];
```

Comment:
What is chain_liquidity_pools and what is chain_liquidity_pools_list and why do we need both?

Reply:
Was doublicated mistakenly, will remove chain_liquidity_pools_list and keep chain_liquidity_pools.